static/front/peek.js

```diff
@@ -528,6 +569,13 @@ $(function() {
 
     $(window).on("hashchange", function() {
       $(document.body).toggleClass("peek", window.location.hash == "#p");
```

Comment:
I'd suggest just selecting document.body once, something like `var docBody = $(document.body);` then using that variable in place of the `$(document.body)` selections.